### PR TITLE
add intellij known issue

### PR DESCRIPTION
**Pull request by mingxiao**
Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

It's best to PR to the most recent relevant branch and leave all the cherry-picking to the
writer, except where earlier branches require factual changes to the content.
For more information about the branches, see https://github.com/pivotal/docs-tap#branches

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -114,6 +114,10 @@ The following issues, listed by area and component, are resolved in this release
 
 This release has the following known issues, listed by area and component.
 
+#### <a id="1-4-0-intellij-ki"></a> Tanzu Developer Tools for Intellij
+
+- If a Workload is deployed via Live Update onto namespace `myNamespace`, then the user needs to set `myNamespace` as the namespace of the current context of their kubeconfig. Otherwise if the user Tanzu Debug, then it will re-deploy the Workload.
```
Comment (paulcwarren) [inline release-notes.hbs.md:119]: wording?

`Otherwise, if the user attempts to Tanzu Debug`?

Comment (mingxiao) [reply release-notes.hbs.md:136]: don't see the extra comma as necessary

**Comment by mingxiao** [comment]
@RichardJJG thanks for the reminder, un-drafted it

**Comment by mingxiao** [comment]
this is just for 1.4

**Pull request merged**